marketplace/commercetools-products/src/product-card.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  EntryCard,
+  DropdownList,
+  DropdownListItem,
+  IconButton,
+  CardDragHandle
+} from '@contentful/forma-36-react-components';
+import { Query } from '@apollo/react-components';
+import GET_PRODUCT_BY_SKU from './ct-product-query.graphql';
+
+import { SortableHandle, SortableElement } from 'react-sortable-hoc';
+
+const DragHandle = SortableHandle(function DragHandle() {
+  return <CardDragHandle>Reorder product</CardDragHandle>;
+});
+
+function getValueByLocale(nameAllLocales, locale) {
+  if (!nameAllLocales.length) {
+    return null;
+  }
+
+  let val = nameAllLocales.find(item => item.locale === locale);
+
+  if (val === undefined) {
+    val = nameAllLocales[0];
+  }
+
+  return val.value;
+}
+
+function ProductCard({ sku, locale, removeItem, index, sortable, isDisabled, projectKey }) {
```

Comment:
again here potentially we could split the components into files to avoid having a file over 200+ lines